evalai/utils/submissions.py

```diff
@@ -0,0 +1,54 @@
+import requests
+import sys
+
+from beautifultable import BeautifulTable
+from click import echo, style
+
+from evalai.utils.auth import get_request_header
+from evalai.utils.common import validate_token
+from evalai.utils.urls import URLS
+from evalai.utils.config import API_HOST_URL, EVALAI_ERROR_CODES
+
+
+def pretty_print_my_submissions_data(submissions):
+    """
+    Funcion to print the submissions for a particular Challenge.
+    """
+    table = BeautifulTable()
+    attributes = ["participant_team_name", "method_name", "execution_time", "status", "submitted_at"]
+    columns_attributes = ["Participant Team", "Method Name", "Execution Time(sec)", "Status", "Submitted At"]
```

Comment:
Two more issues @isht3 I noticed just now
<img width="573" alt="screen shot 2018-06-30 at 2 14 41 pm" src="https://user-images.githubusercontent.com/12206047/42123453-f92fdad0-7c6f-11e8-93d1-d188c7c41bab.png">
1. If there is blank in the `Method Name` from API then ideally we should display None  as we do it on UI.
2. The format for date-time in Submitted At should be `Jun 30, 2018 11:29:42 AM` for some date.

Reply:
@isht3 Also add a field ID in the table that will be used in #56

Reply:
@RishabhJain2018 @varunagrawal I would probably have to scrap the lambda design if I do this. That's okay, right?